cli/addressbook/address.go

```diff
@@ -0,0 +1,107 @@
+package addressbook
```

Comment:
Looking at this again, do we really need the separation between `cli/addressbook` and `config/addressbook`? For wallet accounts it makes sense as you can have different account backends but for the address book you probably won't have different address backends and you can just greatly simplify this and have it just in the config directly? So the whole `cli/addressbook` is not needed and the `EthAddress` field can just go directly into the config.